doc/user/content/sql/create-views.md

```diff
@@ -29,14 +29,14 @@ _new_view_name_  | Optional. The name for the new view. If unspecified, the name
 
 ### Postgres schemas
 
-For Postgres sources, `CREATE VIEWS` will attempt to create each upstream table in the same schema as Postgres. If the publication contains tables` "public"."foo"` and `"otherschema"."foo"`, `CREATE VIEWS` is the equivalent of
+For Postgres sources, `CREATE VIEWS` will attempt to create each upstream table in the same schema as Postgres using the snapshot of publication metadata taken when the source was created. If the source contains tables` "public"."foo"` and `"otherschema"."foo"`, `CREATE VIEWS` is the equivalent of
 
 ```
 CREATE VIEW "public"."foo";
 CREATE VIEW "otherschema"."foo";
 ```
 
-Therefore, in order for `CREATE VIEWS` to succeed, all upstream schemas included in the publication must exist in Materialize as well, or you must explicitly specify the downstream schemas and rename the resulting tables.
+Therefore, in order for `CREATE VIEWS` to succeed, all upstream schemas included in the source must exist in Materialize as well, or you must explicitly specify the downstream schemas and rename the resulting tables.
```

Comment:
I wouldn't really touch this page, since these changes kind of bleed implementation details of the source into a step that is post-source creation. Subbing "publication" for "source" can make the content unnecessarily harder to understand (IIUC, you switched the terms up to hint that what exists at source creation is what's considered rather than what might be in the publication at any given point, but in this case I don't think it's relevant to go into that level of detail).

Reply:
I will be guided by your experience here, my concern was that if we continue to reference publication and someone is trying to figure out why a table they added recently to the publication isn't showing up in `CREATE VIEWS` output they would have no reason to think its related to this new feature in the source.

Reply:
Let's leave it at that and change it if confusion ensues, WDYT?